src/Native/Unix/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ set(VERSION_FILE_PATH "${CMAKE_BINARY_DIR}/../../version.c")
 
 add_compile_options(-Wno-format-nonliteral)
 add_compile_options(-Wno-disabled-macro-expansion)
+add_compile_options(-Wthread-safety-analysis)
```

Comment:
The option to enable that warning is actually `-Wthread-safety` based on clang doc.
Also, could you please move it out of the `-Wno-xxxx` block here which are warning disabling macros?

Reply:
See https://clang.llvm.org/docs/ThreadSafetyAnalysis.html